__tests__/app.test.js

```diff
@@ -258,4 +258,52 @@ describe('/api/articles', () => {
 
     })
 
+    describe('POST /api/articles/:article_id/comments', () => {
+
+        test('A comment is added to the appropriate article', () => {
+
+            return request(app)
+                .post('/api/articles/2/comments')
+                .send({ username: 'lurker' , body: 'Comment'})
+                .expect(201)
+                .then((res) => {
+
+                    expect(res.body).toEqual({ commentAdded : 'Comment' })
+
+                })
+
```

Comment:
I believe the aim for this ticket is to have the API respond with the comment that was just _inserted_ into the table, this can be a good opportunity to use _RETURNING *_ in your database query